scripts/anvil_tools/remove_users_from_project.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+from pandas import *
+import requests
+import urllib.parse
+from oauth2client.client import GoogleCredentials
+
+
+# function to get authorization bearer token for requests
+def get_access_token():
+    """Get access token."""
+
+    scopes = ["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"]
+    credentials = GoogleCredentials.get_application_default()
+    credentials = credentials.create_scoped(scopes)
+
+    return credentials.get_access_token().access_token
+
+
+def remove_users_from_project(project, email_list, verbose):
+    """Adding list of users' emails to the project"""
+    if verbose:
+        print(email_list)
+
+    # Get access token and and add to headers for requests.
+    # -H  "accept: */*" -H  "Authorization: Bearer [token] -H "Content-Type: application/json"
+    headers = {"Authorization": "Bearer " + get_access_token(), "accept": "*/*", "Content-Type": "application/json"}
+    
+    # Stopping running code at the 6th error to avoid multiple failures
+    error_count = 0
+
+    for email in email_list:
+        # URL Encoding the email
+        url_encode_email = urllib.parse.quote(email)
+
+        # Library/putLibraryMetadata
+        uri = f"https://rawls.dsde-prod.broadinstitute.org/api/billing/v2/{project}/members/user/{url_encode_email}"
+
+        # capture response from API and parse out status code
+        response = requests.delete(uri, headers=headers)
```

Comment:
if this code is identical to the other script except for this line, I'd just make it one script (like `modify_users_on_project.py` and have a flag for adding versus removing users to the project

Reply:
or at least refactor all the repeated code into reusable functions